packages/landing/src/components/debug.tsx

```diff
@@ -455,7 +456,13 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
 
   getSourcesIds(type: string): string[] {
```

Comment:
NIT: would be good to document what `type` is for the caller, eg `raster-dem` or `raster`

Reply:
Might just leave it, as the function is very simple to read .

Reply:
One point of comments is so you don't have to read the function to understand what the argument is, especially with generic variable names like "type".

I also wonder about typing it as "raster | raster-dem" as no other "type" makes any sense.

Reply:
typing it as "raster | raster-dem" sounds better option. I will hack that in next PR.